cookbook/integrations/kubernetes/k8s_spark/README.rst

```diff
@@ -5,20 +5,64 @@ Spark on Kubernetes
 
 Flyte can execute Spark jobs natively on a Kubernetes Cluster, which manages the lifecycle, spin-up, and tear down
 of a virtual cluster. It leverages the open-sourced `Spark On K8s Operator <https://github.com/GoogleCloudPlatform/spark-on-k8s-operator>`_
-and can be enabled without signing up for any service. 
+and can be enabled without signing up for any service. This is like running an ``transient cluster`` which are better for production workloads, but have an extra cost of setup and teardown.
```

Comment:
LGTM mostly. Maybe add a line on what a transient cluster here is i.e. brining up driver/executor pods on-demand for each spark job on k8s just to clarify.